tests/e2e/api/digitalAsset.test.ts

```diff
@@ -128,31 +128,31 @@ describe("DigitalAsset", () => {
   });
 
   test("it freezes transfer ability", async () => {
-    const transaction = await aptos.freezeDigitalAssetTransaferTransaction({
+    const transaction = await aptos.freezeDigitalAssetTransferTransaction({
```

Comment:
need to also change it on the api file https://github.com/aptos-labs/aptos-ts-sdk/blob/main/src/api/digitalAsset.ts#L328-L345

And should also marked as a breaking change on CHANGELOG